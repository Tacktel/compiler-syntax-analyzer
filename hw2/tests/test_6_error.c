int main(int a, int b) {

    while ((a == b)) {
        a = b;
    }

    return 1;
}