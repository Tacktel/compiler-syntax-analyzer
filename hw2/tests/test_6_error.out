INT:int
WSPACE
ID:main
LPAREN:(
INT:int
WSPACE
ID:a
COMMA:,
WSPACE
INT:int
WSPACE
ID:b
RPAREN:)
WSPACE
LSCOPE:{
WSPACE
WHILE:while
WSPACE
LPAREN:(
LPAREN:(
ID:a
WSPACE
COMP:==
WSPACE
ID:b
RPAREN:)
RPAREN:)
WSPACE
LSCOPE:{
WSPACE
ID:a
WSPACE
ASSIGN:=
WSPACE
ID:b
TERM:;
WSPACE
RSCOPE:}
WSPACE
RETURN:return
WSPACE
INTVAL:1
TERM:;
WSPACE
RSCOPE:}
